core/src/main/scala/water/support/ModelSerializationSupport.scala

```diff
@@ -54,10 +56,22 @@ trait ModelSerializationSupport {
     destination
   }
 
-  def exportMOJOModel(model: Model[_, _, _], destination: URI): URI = {
-    val destFile = new File(destination)
-    val fos = new FileOutputStream(destFile)
-    model.getMojo.writeTo(fos)
+  def exportMOJOModel(model: Model[_, _, _], destination: URI,sc: SparkContext): URI = {
+    if (destination.getScheme()=="hdfs") {
+      try {
+        val fs = FileSystem.get(sc.hadoopConfiguration)
+        val output = fs.create(new Path(destination))
+        val os = new BufferedOutputStream(output)
+        model.getMojo.writeTo(os)
+      } catch {
+        case e: IllegalArgumentException =>
+          throw new IllegalArgumentException (s"Error while writing to HDFS (most likely no HDFS is available) in destination $destination : ${e.getMessage}")
```

Comment:
Can you pls wrap this message to 2 lines ? I think now it would fail and line too long during scalaCheck